internal/services/cockpit/helpers_cockpit.go

```diff
@@ -103,3 +105,28 @@ func waitForCockpit(ctx context.Context, api *cockpitv1beta1.API, projectID stri
 		RetryInterval: &retryInterval,
 	}, scw.WithContext(ctx))
 }
+
+func cockpitTokenUpgradeV1SchemaType() cty.Type {
+	return cty.Object(map[string]cty.Type{
+		"id": cty.String,
+	})
+}
+
+func cockpitTokenV1UpgradeFunc(_ context.Context, rawState map[string]interface{}, _ interface{}) (map[string]interface{}, error) {
+	defaultRegion := scw.RegionFrPar
```

Comment:
Can we use something similar to `meta.ExtractRegion(d, m)` as default ? Like we do in regional resources.